graphics/window.py

```diff
@@ -0,0 +1,263 @@
+import pygame
+from math import *
+from pygame.math import Vector2
+from pygame.surface import Surface
+
+from model.hex import *
+from model.map import *
+from model.common import *
+from model.vehicle import *
+from graphics.utils import *
+
+
+def hex_size(width: int, height: int, map_size: int) -> float:
+    # Calculates the size of a hexagon to fit the given screen exactly.
+    #
+    # <param name="width">Width of the screen.</param>
+    # <param name="height">Height of the screen.</param>
+    # <param name="map_size">Size of the map.</param>
+    # Note: flat top orientation is assumed
+
+    return min(
+        width / (3 * map_size + 2),
+        height / (2 * map_size + 1) / sqrt(3)
+    )
+
+
+def hex_center(hex: Hex, size: float) -> Vector2:
+    # Calculates the center of the given hexagon with the given size of hex.
+    #
+    # <param name="hex">Hexagon to calculate the center of.</param>
+    # <param name="size">Size of a hexagon.</param>
+
+    e = Vector2(3 / 2, 0)
+    f = Vector2(sqrt(3) / 2, sqrt(3))
+    coords = Vector2(hex.q, hex.r)
+
+    return Vector2(e.dot(coords), f.dot(coords)) * size
+
+
+def regular_polygon_corners(sides: int, size: float = 1, angle: float = 0) -> list:
+    # Calculates corners of a regular polygon with the given size.
+
+    angle_diff = 2 * pi / sides
+
+    return [
+        size * Vector2(cos(angle + angle_diff * i),
+                       sin(angle + angle_diff * i))
+        for i in range(sides)
+    ]
+
+
+def hex_corners(size: float, angle: float = 0) -> list:
+    # Calculates corners of a hexagon with the given size.
+
+    return regular_polygon_corners(6, size, angle)
+
+
+class HexSurface:
+    # Surface of a single hexagon.
+
+    def __init__(self, surface: Surface, size: float):
+        # <param name="surface">Surface to draw on.</param>
```

Comment:
use doc string

think about a different style of method description (you can find a lot of documentation on this topic)